kong-2.1.1-0.rockspec

```diff
@@ -1,9 +1,9 @@
 package = "kong"
-version = "2.1.0-0"
+version = "2.1.1-0"
```

Comment:
Minor note: in the past we have only bumped the rockspec version at release time (even when new migrations for the next version were added)